modules/sdk-coin-celo/src/lib/transactionBuilder.ts

```diff
@@ -49,7 +48,7 @@ export class TransactionBuilder extends EthTransactionBuilder {
       super.loadBuilderInput(tx.toJson());
     } else {
       const txData = JSON.parse(rawTransaction);
-      tx = new Transaction(this._coinConfig, this._common, txData);
+      tx = new Transaction(this._coinConfig, txData);
```

Comment:
https://github.com/BitGo/BitGoJS/blob/81fc641382ce5bab5d3ce05725a09311e4d0a73c/modules/abstract-eth/src/lib/transaction.ts#L54

Seems like Transaction from abstractEth expect this._common